src/main/java/de/doubleslash/keeptime/REST_API/mapper/ColorMapper.java

```diff
@@ -0,0 +1,30 @@
+package de.doubleslash.keeptime.REST_API.mapper;
+
+import de.doubleslash.keeptime.REST_API.DTO.ColorDTO;
+import javafx.scene.paint.Color;
+import org.mapstruct.Mapper;
+
+@Mapper
+public interface ColorMapper {
+   default ColorDTO colorToColorDTO(Color color) {
+      if (color == null) {
+         return null;
+      }
+
+      ColorDTO colorDTO = new ColorDTO();
+      colorDTO.setRed(Color.RED.getRed());
+      colorDTO.setGreen(color.getGreen());
+      colorDTO.setBlue(color.getBlue());
+      colorDTO.setOpacity(color.getOpacity());
+
+      return colorDTO;
```

Comment:
while I do like that you found a way to convert between the colors like this - I would prefer to use the same conversion mechanism like we have when writing the color to the database.

As I think using the hex-code format for colors is more common than floating-point numbers for each rgba.